loglist2/logfilter_test.go

```diff
@@ -130,3 +131,59 @@ func TestRootCompatible(t *testing.T) {
 		})
 	}
 }
+
+// stripErr is helper func for wrapping time.Parse
+func stripErr(t time.Time, _ error) time.Time {
```

Comment:
Best to panic or `log.Fatal()` if there's an error here, so it isn't silently swallowed.

Reply:
Done.